day3/README.md

```diff
@@ -1,8 +1,14 @@
 ## Instructions
 
+Today we will complete our Memory Game app. For the basic setup of the game that we did last time, jsbin was a good platform. 
+However, as our code base will be growing today, having all our code in one single file becomes more of an obstacle.
+Thus, we have created an npm project with the React code from last time.  
```

Comment:
`npm project` høres litt rart ut. Kan kanskje si noe á la `... we have created a real-world project strucutre ...` eller lignende?